btf/strings.go

```diff
@@ -63,6 +71,13 @@ func splitNull(data []byte, atEOF bool) (advance int, token []byte, err error) {
 	return i + 1, data[:i], nil
 }
 
+func lookupString(offset uint32, rawStringTable, baseStringTable *stringTable) (string, error) {
```

Comment:
This should be folded into `stringTable.Lookup`, see https://github.com/cilium/ebpf/pull/691#discussion_r894622433.